Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Entries/RootAssetBrowserEntry.cpp

```diff
@@ -7,8 +7,10 @@
  */
 #include <AzCore/std/containers/vector.h>
 #include <AzCore/Asset/AssetTypeInfoBus.h>
+#include <AzCore/Settings/SettingsRegistryVisitorUtils.h>
```

Comment:
The SettingsRegistry VisitorUtils functions are unused in this file and this include should be removed.

Reply:
Removed